config.json

```diff
@@ -835,6 +835,17 @@
       ]
     },
     {
+      "uuid": "e348a307-078c-5280-65af-a159283d4e79438b755",
+      "slug": "forth",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 10,
```

Comment:
Not sure why difficulty is set to maximum value, it should be lower

Reply:
Difficulty lowered to 5.